server/aesdsocket.c

```diff
@@ -95,7 +102,7 @@ void signal_handler(int signo)
 		printf("Caught signal %d\n", signo);
 		close(fd);
 		close(sockfd);
-		remove("/var/tmp/aesdsocketdata");
+		remove(STORE_IN_THIS_FILE);
```

Comment:
@srki3050 probably won't want to remove the driver node here in the case that we are using the driver endpoint.